# Linter Rule: Disallow extra newlines

**Rule:** `erb-no-extra-newline`

## Description

TODO

## Rationale

TODO

## Examples

### âœ… Good

```html
line 1

line 3
```

### ðŸš« Bad

```erb
line 1


line 3
```

## References

- [Inspiration: ERB Lint `ExtraNewline` rule](https://github.com/Shopify/erb_lint/blob/main/README.md)
