# Disallow adding extra new lines

## Rule: `erb-extra-new-line`

Disallow adding extra new lines

## Examples

### ❌ Incorrect

```erb
line 1


line 3
```

### ✅ Correct

```erb
line 1

line 3
```

## Configuration

This rule has no configuration options.
