# Linter Rule: Require `autocomplete` attribute on `<input>` tags

**Rule:** `html-input-require-autocomplete`

This linter prevents the usage of certain types of HTML `<input>` without an `autocomplete` argument: `color`, `date`, `datetime-local`, `email`, `month`, `number`, `password`, `range`, `search`, `tel`, `text`, `time`, `url`, or `week`.

## Examples

### ❌ Incorrect

```erb
<!-- TODO: Add incorrect example -->
```

### ✅ Correct

```erb
<!-- TODO: Add correct example -->
```

## Configuration

This rule has no configuration options.

## References

* [W3C: Attribute Autocomplete](https://www.w3schools.com/tags/att_autocomplete.asp)
* [MDN: HTML attribute: autocomplete](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/autocomplete)
* [W3C: HTML `<input>` autocomplete tag](https://www.w3schools.com/tags/att_input_autocomplete.asp)
* [W3C: Accessibility Autocomplete](https://www.w3schools.com/accessibility/accessibility_autocomplete.php)
* [WCAG: Technique H98](https://www.w3.org/WAI/WCAG21/Techniques/html/H98)
* [HTML Specification: Autofill](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill)
